### ISSUE TITLE
Error during copy - Unexpected key 'LatestStreamArn' found in params

### ISSUE THREAD

**Issue by mbonig** [enGMzizo/copy-dynamodb-table#3]
I'm trying to copy a table from one account to another. My setup:

```
var copy = require('copy-dynamodb-table').copy
 

var sourceAWSConfig = {
  accessKeyId: 'AK...7Q',
  secretAccessKey: 'at...BJ',
  region: 'us-east-1'
}
 
var destinationAWSConfig = {
  accessKeyId: 'AK...YA',
  secretAccessKey: 'A0...eZ',
  region: 'us-east-1' // support cross zone copying
}
 
copy({
    config: sourceAWSConfig,
    source: {
      tableName: 'Opportunities', // required
      config: sourceAWSConfig // optional , leave blank to use globalAWSConfig
    },
    destination: {
      tableName: 'Opportunities', // required
      config: destinationAWSConfig // optional , leave blank to use globalAWSConfig
    },
    log: true,// default false
    create : true // create destination table if not exist
  },
  function (err, result) {
    if (err) {
      console.log('error:', err)
    }
    console.log('result:', result)
  })
```

produces an error:

```error: { MultipleValidationErrors: There were 2 validation errors:
* UnexpectedParameter: Unexpected key 'LatestStreamLabel' found in params
* UnexpectedParameter: Unexpected key 'LatestStreamArn' found in params
    at ParamValidator.validate (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/param_validator.js:40:28)
    at Request.VALIDATE_PARAMETERS (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/event_listeners.js:125:42)
    at Request.callListeners (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/sequential_executor.js:105:20)
    at callNextListener (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/sequential_executor.js:95:12)
    at /Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/event_listeners.js:85:9
    at finish (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/config.js:320:7)
    at /Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/config.js:338:9
    at Credentials.get (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/credentials.js:126:7)
    at getAsyncCredentials (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/config.js:332:24)
    at Config.getCredentials (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/config.js:352:9)
  message: 'There were 2 validation errors:\n* UnexpectedParameter: Unexpected key \'LatestStreamLabel\' found in params\n* UnexpectedParameter: Unexpected key \'LatestStreamArn\' found in params',
  code: 'MultipleValidationErrors',
  errors:
   [ { UnexpectedParameter: Unexpected key 'LatestStreamLabel' found in params
    at ParamValidator.fail (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/param_validator.js:50:37)
    at ParamValidator.validateStructure (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/param_validator.js:77:14)
    at ParamValidator.validateMember (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/param_validator.js:88:21)
    at ParamValidator.validate (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/param_validator.js:34:10)
    at Request.VALIDATE_PARAMETERS (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/event_listeners.js:125:42)
    at Request.callListeners (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/sequential_executor.js:105:20)
    at callNextListener (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/sequential_executor.js:95:12)
    at /Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/event_listeners.js:85:9
    at finish (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/config.js:320:7)
    at /Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/config.js:338:9
       message: 'Unexpected key \'LatestStreamLabel\' found in params',
       code: 'UnexpectedParameter',
       time: 2018-02-27T21:55:35.455Z },
     { UnexpectedParameter: Unexpected key 'LatestStreamArn' found in params
    at ParamValidator.fail (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/param_validator.js:50:37)
    at ParamValidator.validateStructure (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/param_validator.js:77:14)
    at ParamValidator.validateMember (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/param_validator.js:88:21)
    at ParamValidator.validate (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/param_validator.js:34:10)
    at Request.VALIDATE_PARAMETERS (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/event_listeners.js:125:42)
    at Request.callListeners (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/sequential_executor.js:105:20)
    at callNextListener (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/sequential_executor.js:95:12)
    at /Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/event_listeners.js:85:9
    at finish (/Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/config.js:320:7)
    at /Users/matthew/projects/anotherprojectname/myprojectname/node_modules/aws-sdk/lib/config.js:338:9
       message: 'Unexpected key \'LatestStreamArn\' found in params',
       code: 'UnexpectedParameter',
       time: 2018-02-27T21:55:35.455Z } ],
  time: 2018-02-27T21:55:35.456Z }
result: { Table:
   { AttributeDefinitions: [ [Object] ],
     TableName: 'Opportunities',
     KeySchema: [ [Object] ],
     ProvisionedThroughput: { ReadCapacityUnits: 20, WriteCapacityUnits: 20 },
     StreamSpecification: { StreamEnabled: true, StreamViewType: 'NEW_AND_OLD_IMAGES' },
     LatestStreamLabel: '2017-07-27T17:13:42.958',
     LatestStreamArn: 'arn:aws:dynamodb:us-east-1:213123123:table/Opportunities/stream/2017-07-27T17:13:42.958' } }
```

**Comment by enGMzizo** [comment]
Thanks for reporting. I solved the problem and released v2.0.7 , please give it a try

**Comment by mbonig** [comment]
Yup, seems good now. Thanks!
